flink-connectors/flink-connector-cassandra/src/main/java/org/apache/flink/batch/connectors/cassandra/CassandraOutputFormatBase.java

```diff
@@ -41,7 +41,7 @@
  * @param <OUT> Type of the elements to write.
  */
 public abstract class CassandraOutputFormatBase<OUT> extends RichOutputFormat<OUT> {
-	private static final Logger LOG = LoggerFactory.getLogger(CassandraOutputFormatBase.class);
+	private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
Please revert changes to this class; this PR is for the InputFormat and shouldn't touch other classes unless absolutely necessary.

Reply:
ok